src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegion.java

```diff
@@ -438,7 +439,7 @@ public SimpleInterval getReadSpanLoc() {
      * @param genomeLoc a non-null genome loc indicating the base span of the bp we'd like to get the reference for
      * @return a non-null array of bytes holding the reference bases in referenceReader
      */
-    private static byte[] getReference(final IndexedFastaSequenceFile referenceReader, final int padding, final SimpleInterval genomeLoc) {
+    private static byte[] getReference(final ReferenceSequenceFile referenceReader, final int padding, final SimpleInterval genomeLoc) {
```

Comment:
The naming is a bit weird, but I think the naming issue is on the `ReferenceSequenceFile` side rather than the `referenceReader`   `ReferenceSequence` file is a a closable reader interface that provides `getSequenceMethods`.